### PR TITLE
Fix assip fetch tag to a working one ;

### PR DESCRIPTION
**Pull request by hoshiryu**
UPDATE the form below to describe your PR.

* **Please check if the PR fulfills these requirements**
- [ ] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

Be aware that the PR request cannot be accepted if it doesn't pass the Continuous Integration tests.

I tried loading some animated models to check the skinning plugin is working.
Actually it is not working well with the current version of assimp (the skeleton animation seems broken) so I propose to stick with the last release tag until we fix this. 
Thus we can advance on more urgent stuff.

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)

* **Other information**:

### CODE REVIEW

File: src/IO/external/CMakeLists.txt [hunk 1]
```diff
@@ -19,7 +19,7 @@ if (RADIUM_IO_ASSIMP)
     # For now, only configure in Debug mode to work with xcode
     ExternalProject_Add(assimp
             GIT_REPOSITORY https://github.com/assimp/assimp.git
-            GIT_TAG origin/master
+            GIT_TAG origin/assimp_5.0_release
```
Comment (dlyr) [inline src/IO/external/CMakeLists.txt:22]:
```suggestion
            GIT_TAG v5.0.1
```

Comment (dlyr) [reply src/IO/external/CMakeLists.txt:22]: need to check the syntax, the idea is to use the release TAG instead of a branch HEAD

Comment (hoshiryu) [reply src/IO/external/CMakeLists.txt:22]: I tried using the tag but the configuration step doesn't pass, neither with a commit tag, only with branches. Or maybe I didn't do it the right way.

Comment (hoshiryu) [reply src/IO/external/CMakeLists.txt]: Ok, it works with this syntax, I must have messed it up last time

**Comment by hoshiryu** [comment]
Ready for review

**Comment by hoshiryu** [comment]
For the other external libs, here is a recap:
 - Core:
   - Eigen moved to gitlab
   - OpenMesh has available tags (e.g. OpenMesh-8.1 ), maybe @dlyr can check it
   - cpplocate has available tags (e.g. v2.2.0 ), maybe @nmellado can check it
 - Engine:
   - glm is already on tag  0.9.9.5, which seems to be the latest
   - glbinding is on commit 663e19cf1 from @dlyr's repo, maybe @dlyr or @MathiasPaulin  can check it
   - globject is on commit 11c559a0 from official repo, maybe @dlyr or @MathiasPaulin  can check it
   - stb in on commit 1034f5e5 from official repo, but no tag available
 - IO:
   - Assimp -> moving from master to tag v5.0.1
   - tinyply is on master branch, maybe @nmellado can check it

**Comment by dlyr** [comment]
Let's move to #540

**Pull request closed without merging**